dashboards/resources/cluster.libsonnet

```diff
@@ -129,7 +129,7 @@ local var = g.dashboard.variable;
         + tsPanel.queryOptions.withTargets([
           prometheus.new(
             '${datasource}',
-            'sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{%(clusterLabel)s="$cluster"}) by (namespace)' % $._config
+            'sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate{%(clusterLabel)s="$cluster"}) by (namespace)' % $._config
```

Comment:
The recording rule says `node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate5m`, but all the code changes say `node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate`.